packages/frontend/src/js/app/pages/Account/index.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+    <PrivateLayout class="mb-16">
+        <LoadingError v-if="error || (!user && !loading)">
+            L'utilisateur n'existe pas en base de données.
```

Comment:
Est-ce qu'il ne faudrait pas afficher l'erreur retournée par l'api ici : `{{ error }}`.
Dans le cas `(!user && !loading)`, modifier `load()` pour générer une erreur après le try/catch : `if (!user) { this.error = "L'utilisateur n'existe pas en base de données."; }`.

Dans ce cas, le `v-if` devient simplement `v-if="error"`